packages/cli/api-importers/conjure/conjure-to-fern/src/ConjureImporter.ts

```diff
@@ -169,7 +169,12 @@ export class ConjureImporter extends APIDefinitionImporter<ConjureImporter.Args>
                             continue;
                         }
                         if (typeof argDeclaration === "string") {
-                            endpoint.request = { body: argDeclaration === "binary" ? "bytes" : argDeclaration };
+                            if (!endpoint.request) {
+                                endpoint.request = {};
+                            } else if (typeof endpoint.request === "string") {
+                                endpoint.request = { body: "" };
+                            }
```

Comment:
I think it make more sense to make this `endpoint.request = { body: endpoint.request }`

Reply:
Fixed